### PR TITLE
Bug 247 equality constraints

### PR DESCRIPTION
**Pull request by Ethan-Russell**

### CODE REVIEW

File: src/results/formulas.jl [hunk 1]
```diff
@@ -322,6 +322,12 @@ end
 
 Base.getproperty(d::DictWrapper, s::Symbol) = getfield(d, :d)[s]
 
+function col_or_container(data, table, s::Symbol)
+    hasproperty(table, s) && return table[!, s]
```
Comment (sallyrobson) [inline src/results/formulas.jl:326]: this is great, thanks for adding it in

File: src/results/parse.jl [hunk 2]
```diff
@@ -122,20 +159,12 @@ Adds power-based results.  See also [`get_table_summary`](@ref) for the below su
 | table_name | col_name | unit | description |
 | :-- | :-- | :-- | :-- |
 | :bus | :pgen | MWGenerated | Average Power Generated at this bus |
-| :bus | :egen | MWhGenerated | Electricity Generated at this bus for the weighted representative hour |
```
Comment (sallyrobson) [inline src/results/parse.jl:125]: why are we removing all of these?

**Review by sallyrobson — APPROVED** [review]
I think this overall looks good but I saw that there are lots of places were you are removing "e____" and replacing with "p___ * hour_weights", could you add an explanation to this PR of why you did that?

File: src/results/parse.jl (hunk 2 above)
Comment (Ethan-Russell) [reply src/results/parse.jl:125]: In short, to save memory and time.  I made several changes in this PR to make the serialized data.jls files smaller.  The egen, elserv, eflow, etc. are all easily computed from their power equivalent.  Choosing not to save `egen_gen` alone saves 12MB from each results file (assuming 30k generators, 52 repr hours).

**Pull request merged**